charts/ravendb-cluster/scripts/create-cluster.sh

```
@@ -3,42 +3,39 @@
 set -e
 
 echo "Installing prerequisites..."
-# prerequisites
 apt-get update
 apt-get install curl unzip jq -y
 
-# copy the package from readonly volume
+
 echo "Copying /ravendb/ravendb-setup-package-readonly/pack.zip to the /ravendb folder..."
 cp -v /ravendb/ravendb-setup-package/*.zip /ravendb/pack.zip
 
-# unzip the pack
-echo "Extracting files from the pack..."
+
+echo "Extracting files from the package..."
 mkdir /ravendb/ravendb-setup-package-copy
 cd /ravendb
 unzip -qq pack.zip -d ./ravendb-setup-package-copy/ > /dev/null
 cd ravendb-setup-package-copy/A
```

'A' might not be the first node right

---

Also using `cd` can easily get you lost. Also a failure to `cd` should result in a fatal error and script failing fast.